backend/battles/tests/test_views.py

```diff
@@ -0,0 +1,51 @@
+from django.test import TestCase, Client
+from django.urls import reverse
+
+from model_mommy import mommy
+
+
+class BattleListViewTests(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+
+    def test_battle_list(self):
+        battle1 = mommy.make('battles.Battle')
+        battle2 = mommy.make('battles.Battle')
+
+        url = reverse('battles:battle_list')
+        response = self.client.get(url)
+
+        self.assertContains(response, battle1.id, status_code=200)
+        self.assertContains(response, battle2.id, status_code=200)
+
+
+class BattleDetailViewTests(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+
+    def test_battle_detail(self):
+        battle = mommy.make('battles.Battle')
+
+        url = reverse('battles:battle_detail', args=[battle.id])
+        response = self.client.get(url)
+
+        self.assertContains(response, battle.id, status_code=200)
+
+
+class BattleCreateViewTests(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+
+    def test_battle_create(self):
```

Comment:
There's something that we are missing here. As commented bellow this test should be failing since there's no user with the email "jaenia@..."

Reply:
This test should also assert if the new item is in the database